### DISCUSSION TITLE
Connect rigid bodies to particles via springs?

### DISCUSSION THREAD

**Discussion by touste** [NVIDIA/warp#316] [Q&A]
Hi, is there any way to connect rigid bodies to particle-based structures via springs? In the examples I only found particle-based simulations (ex soft body, cloth, spring cage) or rigid body systems (ex quadrupled, chain).
I'm interested in using warp for multibody simulations in biomechanics where systems are based on rigid bodies (bones), joints, activation units (muscles) and spring connectors (ligaments).
Thanks!

**Comment by eric-heiden** [comment]
Continuing in https://github.com/NVIDIA/warp/issues/380
